src/new/templates/rust/ui/framework-process/framework-process/src/lib.rs

```diff
@@ -0,0 +1,69 @@
+use hyperprocess_macro::hyperprocess;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
```

Comment:
Prefer grouping args like

stdlib

imported in Cargo.toml

and possibly a third group of "written by us"

with each group alphabetized.

So here should either be:

```rs
use std::collections::HashMap;

use hyperprocess_macro::hyperprocess;
use serde::{Deserialize, Serialize};
```

or

```
use std::collections::HashMap;

use serde::{Deserialize, Serialize};

use hyperprocess_macro::hyperprocess;
```